packages/vincent-tools/vincent-tool-uniswap-swap/test/e2e/helpers/check-mint-fund-pkp.ts

```diff
@@ -0,0 +1,88 @@
+import { formatEther } from 'viem';
```

Comment:
Feels like we need to make an `e2e-vincent-tools` package that contains all of the boilerplate for PKP / app management, and imports these tools and runs them - this is a lot of duplicated code.  We can land this as-is but we should circle back on this after we've shipped things.